src/Tribe/Admin/Notice/Install_Event_Tickets.php

```diff
@@ -0,0 +1,436 @@
+	 */
+	public function is_installed( $slug = '' ): bool {
+		$installed_plugins = get_plugins();
+
+		return array_key_exists( $this->get_plugin_path( $slug ), $installed_plugins ) || in_array( $this->get_plugin_path( $slug ), $installed_plugins, true );
+	}
+
+	/**
+	 * Checks if `Event Tickets` is active.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $slug The plugin slug.
+	 *
+	 * @return boolean True if active.
+	 */
+	public function is_active( $slug = '' ): bool {
+		return is_plugin_active( $this->get_plugin_path( $slug ) );
+	}
+
+	/**
+	 * Check if we're on the classic "Install Plugin" page.
+	 *
+	 * @since TBD
+	 *
+	 * @return boolean
+	 */
+	public function is_install_plugin_page(): bool {
+		return 'install-plugin' === tribe_get_request_var( 'action' );
+	}
+
+	/**
+	 * Should the `Install` notice be displayed?
+	 *
+	 * @since TBD
+	 *
+	 * @return bool True if the install notice should be displayed.
+	 */
+	public function should_display_notice_install() {
+		return ! $this->is_installed()
+			&& empty( tribe_get_request_var( 'welcome-message-the-events-calendar' ) )
+			&& ! $this->is_install_plugin_page();
+	}
+
+	/**
+	 * Should the `Activate` notice be displayed?
+	 *
+	 * @since TBD
+	 *
+	 * @return bool True if the activate notice should be displayed.
+	 */
+	public function should_display_notice_activate() {
+		return $this->is_installed() && ! $this->is_active() && ! $this->is_install_plugin_page();
+	}
+
+	/**
+	 * Install notice for `Event Tickets`.
+	 *
+	 * @since TBD
+	 */
+	public function notice_install() {
+		if ( ! $this->should_display_notice_install() ) {
+			return '';
+		}
+
+		$this->enqueue_assets();
+
+		$html = $this->get_template()->template(
+			'notices/install-event-tickets',
+			$this->get_template_data(),
+			false
+		);
+
+		tribe_notice(
+			'event-tickets-install',
+			$html,
+			[
+				'dismiss' => true,
+				'type'    => 'warning',
+			]
+		);
+	}
+
+	/**
+	 * Enqueue assets required for the notice.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	private function enqueue_assets() {
+		wp_enqueue_style( 'wp-components' );
+		tribe_asset_enqueue_group( self::$assets_group );
+	}
+
+	/**
+	 * Activate notice for `Event Tickets`.
+	 *
+	 * @since TBD
+	 */
+	public function notice_activate() {
+		if ( ! $this->should_display_notice_activate() ) {
+			return '';
+		}
+
+		$this->enqueue_assets();
+
+		$args = [
+			'description'  => __( 'You\'re almost there! Activate Event Tickets to manage attendee registration and ticket sales to your events, for free.', 'the-events-calendar' ),
+			'button_label' => __( 'Activate Event Tickets', 'the-events-calendar' ),
+			'action'       => 'activate',
+		];
+
+		$html = $this->get_template()->template(
+			'notices/install-event-tickets',
+			$this->get_template_data( $args ),
+			false
+		);
+
+		tribe_notice(
+			'event-tickets-activate',
+			$html,
+			[
+				'dismiss' => true,
+				'type'    => 'warning',
+			]
+		);
```

Comment:
Same problems as the notice install. Not leveraging `tribe_notice` or `tribe_asset` tools to avoid early rendering and conditionals.